src/main/java/io/github/minecraftcursedlegacy/api/vanillachecker/VanillaChecker.java

```diff
@@ -0,0 +1,18 @@
+package io.github.minecraftcursedlegacy.api.vanillachecker;
+
+import java.util.HashMap;
+
+import net.minecraft.entity.player.Player;
+
+public class VanillaChecker {
+    private VanillaChecker(){/*No Constructor*/}
+
+    /**
+     * Don't Touch!
+     */
+    public static HashMap<String, Boolean> playermap = new HashMap<String, Boolean>();
```

Comment:
can you... implement it a better way than this? Preferably split up the API and Implementation into separate packages

Reply:
And use methods in the impl to only expose specific parts.